src/main/java/com/cinema/servlets/MainServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.cinema.servlets;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.Instant;
+
+
+public class MainServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+        // if you remove line with date, you change nothing.
+        response.addDateHeader("Date", Instant.now().toEpochMilli());
+
+        response.setContentType("text/html");
+        PrintWriter out = response.getWriter();
+        out.println("<p>Hello World!</p>");
```

Comment:
why is it here at all?

Reply:
That there was at least something on the page. I removed that